.github/scripts/snapshots.js

```diff
@@ -0,0 +1,215 @@
+const fs = require("fs");
+const path = require("path");
+
+// Constants
+const RELEASE_TAG = "nightly";
+const DAYS_TO_KEEP = 3;
```

Comment:
We keep snapshots for 3 days to ensure snapshot consumers won't use a deleted snapshot. Imagine the following scenario:

1. The job looks up the latest mlflow-skinny snapshot URL.
2. The latest mlflow-skinny snapshot is removed for update.
3. The job uses the snapshot but it doesn't exist since it's being updated.